ml-algorithms/build.gradle

```diff
@@ -67,21 +67,21 @@ dependencies {
         }
     }
 
-    implementation platform('software.amazon.awssdk:bom:2.29.12')
-    api 'software.amazon.awssdk:auth:2.29.12'
+    implementation platform('software.amazon.awssdk:bom:2.30.18')
+    api 'software.amazon.awssdk:auth:2.30.18'
     implementation 'software.amazon.awssdk:apache-client'
     implementation ('com.amazonaws:aws-encryption-sdk-java:2.4.1') {
         exclude group: 'org.bouncycastle', module: 'bcprov-ext-jdk18on'
     }
     implementation 'org.bouncycastle:bcprov-jdk18on:1.78.1'
 
     compileOnly group: 'software.amazon.awssdk', name: 'aws-core', version: '2.30.18'
-    compileOnly group: 'software.amazon.awssdk', name: 's3', version: '2.29.12'
-    compileOnly group: 'software.amazon.awssdk', name: 'regions', version: '2.29.12'
+    compileOnly group: 'software.amazon.awssdk', name: 's3', version: '2.30.18'
```

Comment:
Does core uses `awssdk`? I was wondering if we could use something like `implementation("com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}")` so that we don't have to worry about version from plugin level?

Reply:
Core did have a pointer here: https://github.com/opensearch-project/OpenSearch/blob/main/gradle/libs.versions.toml#L55

However, netty has its own pointer so they just updated netty directly:
https://github.com/opensearch-project/OpenSearch/commit/c82fffe8e333a8fff600dc22fa2e1aed7276d180

Reply:
Will update core to the new version first.